Source/ui_unix/cdevice.cpp

```diff
@@ -0,0 +1,164 @@
+#include "cdevice.h"
```

Comment:
Not quite sure about that file's name. I think something like `VfsDevice.cpp/.h` would be more fitting.